### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by heintz427**
Merging several documentation updates to resolve the following issues: 

https://github.com/opentap/opentap/issues/321:
- Updated \opentap\CONTRIBUTING.md to fix broken link.

https://github.com/opentap/opentap/issues/217: 
- Updated \opentap\LICENSE.txt file to update Copyright date/year.

https://github.com/opentap/opentap/issues/1462: 
- Updated \opentap\doc\Developer Guide\Introduction\Readme.md to update to VS 2022. 

https://github.com/opentap/opentap/issues/1452:  
- Updated \opentap\doc\User Guide\Introduction\Readme.md to add note about parent steps.

### CODE REVIEW

File: doc/User Guide/Introduction/Readme.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ verdict is encountered, and continue otherwise. The following break conditions c
 
 Specifying break conditions overrides the engine settings in the given step, and all child steps of that step. Child
 steps can further override the value set by their ancestor if necessary. When a break condition is encountered, execution
-of all child steps is stopped, and the test plan resumes from the point where the break was encountered.
+of all child steps is stopped, and the test plan resumes from the point where the break was encountered (break conditions also affect the parent step).
```
Comment (alnlarsen) [inline doc/User Guide/Introduction/Readme.md:92]: The technical content of this section needs a bigger revision to make the behavior clear. Can you remove this change? Then I will figure out what we should actually say here in collaboration with Rolf and Witold

**Review by alnlarsen — CHANGES_REQUESTED** [review]
Looks good, minor point about break conditions

File: doc/User Guide/Introduction/Readme.md (hunk 1 above)
Comment (heintz427) [reply doc/User Guide/Introduction/Readme.md:92]: Hi @alnlarsen. I have removed the change here.

**Review by alnlarsen — APPROVED** [review]
Looks good, let's merge!

**Comment by heintz427** [comment]
Merge changes

**Pull request merged**